extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -99,6 +99,14 @@ tags: {  # <amp-story>
     name: "title"
     mandatory: true
   }
+  attrs: {
+    name: "live-story"
+    value: ""
+  }
+  attrs: {
+    name: "disabled"
```

Comment:
I'm not sure this semantically makes sense; this sounds like the story itself is somehow disabled.

How hard would it be to make this a value of the `live-story` attribute (i.e. `live-story="disabled"`)?

Reply:
+1, or `live-story-disabled`?

Reply:
Opened #22879